planner/utils/google_utils.py

```diff
@@ -0,0 +1,79 @@
+"""
+Functions for getting attributes about points of interest.
+Extracts data from POI locations (elevation, safety, etc.)
+
+Uses: Google Place Search
+
+Uses the python-google-places wrapper:
+https://github.com/slimkrazy/python-google-places
+"""
+
+from googleplaces import GooglePlaces, types
+from configparser import ConfigParser
+import psycopg2
+import json
+
+class GoogleHelper:
+	"""
+	Helper class for accessing the google API
+	"""
+	def __init__(self, gmaps_api_key=None, conn=None):
+		import db_conn
+		if gmaps_api_key is None:
+			config = ConfigParser()
+			with open('config.json') as f:
+			    config = json.load(f)
+			gmaps_api_key = config["gmapsApiKey"]
+		if conn is None:
+			conn = db_conn.connPool.getconn()
+
+		self.conn = conn
+		self.cur = conn.cursor()
+		self.google_places = GooglePlaces(config["gmapsApiKey"])
+
+
+	def save_to_json(self, data, file_name='output.txt'):
+		with open(file_name, 'w') as outfile:
+			json.dump(data, outfile)
+
+
+	def get_pois(self, lat_lng, radius=15, type_list=['park']):
```

Comment:
Default arg `['park']` is mutable. It's harmless now but might lead to bugs in the future.

docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html